tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -0,0 +1,240 @@
+            const stdoutSandboxIsNotRunning = execSync("taq stop sandbox local", {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+            expect(stdoutSandboxIsNotRunning).toEqual("The local sandbox was not running.");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+
+    // TODO: Currently it cannot be done until this issue has been resolved
+    // https://github.com/ecadlabs/taqueria/issues/243
+    test.skip('Verify that taqueria flextesa plugin can start and stop a sandbox with custom name', async () => {
+        try {
+            // Setting up docker container name
+            dockerName = "test"
+
+            // 1. Updating config to add test sandbox
+            execSync(`cp e2e/data/config-flextesa-test-sandbox.json ${taqueriaProjectPath}/.taq/config.json`);
+
+            // 2. Run sandbox start command with customer name
+            const stdoutStart = execSync(`taq start sandbox ${dockerName}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 3. Verify that sandbox has been started and taqueria returns proper message into console
+            expect(stdoutStart).toEqual("Started test.");
+
+            // 4. Run list accounts command and verify that it returns list of default accounts
+            await waitForExpect(() => {
+                const stdoutList = execSync(`taq list accounts ${dockerName}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+                expect(stdoutList).toContain("ken");
+                expect(stdoutList).not.toContain("tran");
+            });
+
+            // 5.  Run stop command and verify the output
+            const stdoutStop = execSync(`taq stop sandbox ${dockerName}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 6. Verify that taqueria returns proper message into console
+            expect(stdoutStop).toEqual("Stopped test.");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+
+    // TODO: Currently it cannot be done until this issue has been resolved
+    // https://github.com/ecadlabs/taqueria/issues/243
+    test.skip('Verify that taqueria flextesa plugin can retrieve data from updated config after restart', async () => {
+        try {
+            // Setting up docker container name
+            dockerName = "local"
+
+            // 1. Start sandbox
+
+            // 2. Check balance
+
+            // 3. Stop sandbox
+
+            // 4. Update config
+
+            // 5. start sandbox again
+
+            // 6. Check balance again and see that it changed
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+
+    // Clean up process to stop container if it was not stopped properly during the test
+    afterEach( () => {
+        try {
+            let [,dockerContainer] = execSync(`docker ps --filter "name=${dockerName}"`).toString().trim().split(/\r?\n/);
```

Comment:
The comma looks like a syntax typo.

Line 30 and 64 appear to have the same line. We should probably follow the DRY rule here and extract this to a function called findContainerByName or something like that.

Reply:
We use should an underscore for the variable placeholders which read a little better. This is recommended in the TypeScript Handbook:
https://www.typescriptlang.org/docs/handbook/release-notes/overview.html#destructured-variables-can-be-explicitly-marked-as-unused